ani-cli

```diff
@@ -352,14 +352,20 @@ history)
 *)
 	while [ -z "$query" ]; do
 		if [ "$use_external_menu" = "0" ]; then
+			clear
 			printf "Search anime: " && read -r query
 		else
 			query=$(: | external_menu "" "Search anime:")
 		fi
 	done
 	query=$(printf "%s" "$query" | sed "s| |%20|g")
 	anime_list=$(search_anime "$query")
-	[ -z "$anime_list" ] && die "No results found!"
+	while [ -z "$anime_list" ]; do
+		clear
```

Comment:
There is an escape sequence that sets the cursor to the previous line and deletes it. Use that, not clear.